README.md

```diff
@@ -9,30 +13,51 @@ The Azure Artifacts Credential Provider automates the acquisition of credentials
 -   [Use](#use)
 -   [Session Token Cache Locations](#session-token-cache-locations)
 -   [Environment Variables](#environment-variables)
--   [Release version 1.0.0](#release-version-1.0.0)
--   [Upcoming version 2.0.0](#release-version-2.0.0)
+-   [Release version 1.0.0](#release-version-100)
+-   [Upcoming version 2.0.0](#release-version-200)
 -   [Help](#help)
 -   [Contribute](#contribute)
 
 ## Prerequisites
 
-### MSBuild on Windows
+If using one of the credential provider tools for an environment other than .NET (npm, Maven, etc.), follow the guidance for that tool. A separate installation of the Azure Artifacts Credential Provider is not required.
 
-Install [Visual Studio version 15.9-preview1 or later](https://visualstudio.microsoft.com/vs/preview/) to get the required version of MSBuild (`15.8.166.59604` or later). Alternatively, you can download MSBuild directly by downloading the [Build Tools for Visual Studio](https://visualstudio.microsoft.com/downloads/?q=build+tools). MSBuild is also installed as a part of the [.NET Core SDK](https://dotnet.microsoft.com/download).
+### For `dotnet` (preferred)
```

Comment:
can we remove `(preferred)` and confirm the dotnet nuget command version that supports  nuget plugins as a .NET tool